dev/integration_tests/web/web/index_with_flutterjs_custom_sw_version.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE HTML>
+<!-- Copyright 2014 The Flutter Authors. All rights reserved.
+Use of this source code is governed by a BSD-style license that can be
+found in the LICENSE file. -->
+<html>
+<head>
+  <meta charset="UTF-8">
+  <meta content="IE=Edge" http-equiv="X-UA-Compatible">
+
+  <title>Integration test. App load with flutter.js. Custom serviceWorkerVersion provided.</title>
+
+  <!-- iOS meta tags & icons -->
+  <meta name="apple-mobile-web-app-capable" content="yes">
+  <meta name="apple-mobile-web-app-status-bar-style" content="black">
+  <meta name="apple-mobile-web-app-title" content="Web Test">
+  <link rel="manifest" href="manifest.json">
+  <script>
+    // The value below is injected by flutter build, do not touch.
+    var serviceWorkerVersion = null;
+  </script>
+  <!-- This script adds the flutter initialization JS code -->
+  <script src="flutter.js" defer></script>
+</head>
+<body>
+  <script>
+    window.addEventListener('load', function(ev) {
+      // Download main.dart.js
+      _flutter.loader.loadEntrypoint({
+        serviceWorker: {
+          serviceWorkerVersion: '123',
+        }
+      }).then(function(engineInitializer) {
+        return engineInitializer.autoStart();
+      });
```

Comment:
Instead of this then, use the `onEntrypointLoaded callback of the `loadEntrypoint` function. [Docs](https://docs.flutter.dev/platform-integration/web/initialization#engine-configuration-example).

Reply:
This `index_with_flutterjs_custom_sw_version.html` file is based on the `index_with_flutterjs_short.html` file that is used in another test, but I can change the base to `index_with_flutterjs_entrypoint_loaded.html` if you want

Reply:
@p-mazhnik yes, the other examples are also slightly outdated. Just fix this one, so it's something like:

```js
      _flutter.loader.loadEntrypoint({
        serviceWorker: {
          serviceWorkerVersion: '123',
        },
        onEntrypointLoaded: async function(engineInitializer) {
          return engineInitializer.autoStart();
        }
      });
```

Reply:
Done